src/tests/unit/doughs-serializer.test.js

```diff
@@ -0,0 +1,120 @@
+import chai from 'chai';
+import config from 'config';
+import _ from 'lodash';
+import proxyquire from 'proxyquire';
+import sinon from 'sinon';
+
+import { openapi } from 'utils/load-openapi';
+import { doughSerializerData } from './test-data';
+
+const doughAttributes = openapi.definitions.DoughAttributes.properties;
+
+chai.should();
+
+let rawDough;
+let rawDoughs;
+let doughSerializer;
+
+const {
+  nullSpecialInstructionsDough,
+  emptyStringSpecialInstructionsDough,
+} = doughSerializerData;
+
+describe('test doughs serializer', () => {
+  before(() => {
+    sinon.replace(config, 'get', () => ({ oracledb: {} }));
+    // importing doughSerializer normally causes config to become unextensible,
+    // which then breaks other tests. This prevents that problem, even though
+    // stubbing imports is not required.
+    doughSerializer = proxyquire('../../api/v1/serializers/doughs-serializer', {});
+  });
+  after(() => {
+    sinon.restore();
+  });
+  describe('serializeDough', () => {
+    describe('when it gets a dough object with specialInstructions = `null`', () => {
+      beforeEach(() => {
+        rawDough = nullSpecialInstructionsDough;
+      });
+      it('converts specialInstructions to an empty string', () => {
+        const serializedDough = doughSerializer.serializeDough(rawDough);
+        serializedDough.data.attributes.specialInstructions.should.equal('');
+      });
+    });
+    describe('when it gets a dough with specialInstructions = \'\'', () => {
+      beforeEach(() => {
+        rawDough = emptyStringSpecialInstructionsDough;
+      });
+      it('leaves specialInstructions as an empty string', () => {
+        const serializedDough = doughSerializer.serializeDough(rawDough);
+        serializedDough.data.attributes.specialInstructions.should.equal('');
+      });
+    });
+    describe('when it gets a raw dough with a valid format', () => {
+      beforeEach(() => {
+        rawDough = emptyStringSpecialInstructionsDough;
+      });
+      it('adds a top level self link', () => {
+        const serializedDough = doughSerializer.serializeDough(rawDough, 'doughs');
+        serializedDough.links.self.should.equal('/v1/doughs');
+      });
+      it('adds type and id at the top level of data', () => {
+        const serializedDough = doughSerializer.serializeDough(rawDough, 'doughs');
+        serializedDough.data.type.should.equal('dough');
+        serializedDough.data.id.should.equal('201');
+      });
+      it('adds all of the right attributes to `attributes`', () => {
+        const serializedDough = doughSerializer.serializeDough(rawDough, 'doughs');
+        _.forIn(doughAttributes, (value, attribute) => {
+          serializedDough.data.attributes.should.have.property(attribute);
+        });
+        serializedDough.data.attributes.should.not.have.property('id');
+      });
+    });
+  });
+  describe('serializeDoughs', () => {
```

Comment:
Make use of whitespace to make your code more readable. A space between `serializeDough` and `serializeDoughs` tests will help segment the tests for others.